pydatastructs/utils/misc_util.py

```diff
@@ -170,6 +171,33 @@ def __new__(cls, data=None, links=['next'], addrs=[None]):
     def __str__(self):
         return str(self.data)
 
+class SkipListNode(Node):
```

Comment:
Why can't we simply use `LinkedListNode` with number of `links` equal to the number of levels in `SkipList`?

Reply:
`links=['next', 'forward']`, `addrs=[None,[]]` is this what you are suggesting?

Reply:
I would have to look into SinglyLinkedList implementation for that, can we do it later, after RedBlackTree is complete?